src/js/angular/explore/controllers.js

```diff
@@ -579,7 +586,7 @@ function EditResourceCtrl($scope, $http, $location, toastr, $repositories, $uibM
     $scope.$watch(function () {
         return $repositories.getActiveRepository();
     }, function () {
-        if ($scope.activeRepository()) {
+        if ($licenseService.isLicenseValid() && $scope.activeRepository()) {
```

Comment:
Why? Aren't you doing the check in the beginning of the getClassInstancesDetails method already?

Reply:
Yes, I'll fix the duplication.